### DISCUSSION TITLE
Issues connecting with DigitalOcean Apps

### DISCUSSION THREAD

**Discussion by MeisamMulla** [community/community#40105] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I'm having an issue connecting my DigitalOcean account with my GitHub account and was told by DigitalOcean to submit this request here.

I have successfully been able to connect my account in the past and all was working fine until I needed to let DigitalOcean see more of my repositories. When I tried clicking the link from DigitalOcean to GitHub I kept landing on the following unicorn page:

![Screenshot 2022-11-25 at 3 13 03 PM](https://user-images.githubusercontent.com/1777638/204051400-67e6998c-6273-4dbe-a388-40a858194700.png)

I have tried completely deleting the app from GitHub however that just made my existing connections stop working and I can no longer deploy my apps.

I should mention that other apps work just fine. I have tried this from multiple browsers and multiple computers in multiple locations all with the same results.

Any help would greatly be appreciated.

**Comment by ossanna16** [comment]
Hi there @MeisamMulla and welcome to our community! Thank you for reporting this!
